src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs

```diff
@@ -105,6 +108,8 @@ private void convertParsedDataColumn(DataRepository dataRepository, KeyValuePair
             else
                values[i++] = (float) adjustedValue;
          }
+         if (lloqValue != null)
+            dataInfo.LLOQ = Convert.ToSingle(dimension.UnitValueToBaseUnitValue(dimension.FindUnit(lloqValue.Unit), lloqValue.Lloq));
```

Comment:
I think this should really be tested. A core function, no UI.

Reply:
I agree... I will add some test on next week

Reply:
I fixed the ignored tests for the mapper and added a new one for the lloq